bundler/lib/bundler/lockfile_parser.rb

```diff
@@ -226,10 +229,7 @@ def parse_checksum(line)
       platform = $4
 
       version = Gem::Version.new(version)
-      platform = platform ? Gem::Platform.new(platform) : Gem::Platform::RUBY
```

Comment:
why is this line removed?

Reply:
oh, it's duplicating the behavior of Gem::NameTuple.new so it's unnecessary. Maybe it can happen in another PR, but it's something I noticed when going through the code.

Reply:
I should say, not duplicating exactly, it's making a Gem::Platform, then to_s on the platform. The only reason this would matter is if there's an inconsistent platform name saved into the checksums section of the lockfile that gets resolved by passing through `Gem::Platform.new#to_s`. I'll revert it just based on this slim case.